i2c/oled_i2c/img_to_array.py

```diff
@@ -28,17 +27,19 @@
 except OSError:
     raise Exception("Oops! The image could not be opened.")
 
-if im.size[0] > OLED_WIDTH or im.size[1] > OLED_HEIGHT:
+img_width = im.size[0]
+img_height = im.size[1]
+
+if img_width > OLED_WIDTH or img_height > OLED_HEIGHT:
+    print(f'Your image is f{img_width} pixels wide and {img_height} pixels high, but...')
     raise Exception(f"OLED display only {OLED_WIDTH} pixels wide and {OLED_HEIGHT} pixels high!")
```

Comment:
I think it'd be better to put a `\n` in the middle of the long Exception message, rather than splitting the message between `print` and `Exception` :man_shrugging: